### PR TITLE
add yalc setup to scripts

### PR DESCRIPTION
**Pull request by valeriia-ruban**
### :hammer_and_wrench: Description

A description of how we can use [Yalc](https://github.com/wclr/yalc) can be found in the [RFC](https://go.hashi.co/rfc/csl-3822). There isn’t a ton to “add to the repo” except a script for publishing the local build to yalc, and a script for cleanup if it seems like it would be helpful. It would also be good to include documentation on how to use yalc with the toolkit.

**AC**
Add script for publishing builds with yalc (optionally one for cleanup as well)

Add documentation on how to use yalc with the toolkit
### :link: External Links
[RFC](https://go.hashi.co/rfc/csl-3822)
[Yalc](https://github.com/wclr/yalc)
https://hashicorp.atlassian.net/browse/CC-4211

### :building_construction: How to Build and Test the Change
- cd toolkit 
- yarn toolkit:publish
- yarn toolkit:cleanup

### :+1: Definition of Done

- [ ] New functionality works?
- [ ] Tests added?
- [ ] Docs updated?

### :speech_balloon: Using the [Netlify feedback ladder](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify/)

- :mount_fuji: **[mountain]**: Blocking and requires immediate action.
- :moyai: **[boulder]**: Blocking.
- :white_circle: **[pebble]**: Non-blocking, but requires future action.
- :hourglass_flowing_sand: **[sand]**: Non-blocking, but requires future consideration.
- :sparkles: **[dust]**: Non-blocking. "Take it or leave it"

### CODE REVIEW

File: toolkit/package.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js:fix": "eslint . --fix",
     "lint:types": "glint",
+    "toolkit:publish": "yalc publish --content",
+    "toolkit:cleanup": "yalc installations clean @hashicorp/consul-ui-toolkit",
```
Comment (WenInCode) [inline toolkit/package.json:29]: Some thoughts about changing the naming of these. None of the other yarn scripts in the `toolkit/package.json` have the prefix `toolkit` as they are already in the `toolkit`. It also might be good to have the script name be clear that it's not actually publishing to npm.  Maybe something like `publish:local` or `publish:yalc` or something along those lines.

Comment (WenInCode) [reply toolkit/package.json:29]: I don't mind the `toolkit:` prefix for my second suggestion, which is to add scripts to the root `package.json` to run these from the root. 

Example:
```
// in the root package.json scripts
"toolkit:publish:local": "yarn workspace @hashicorp/consul-ui-toolkit run toolkit:publish"
```

That way you can publish and cleanup from the root. Keeping the prefix here would make more sense as it lets you know where the command is being run.. however the other ones either use a post fix (`start:documentation`) or don't have either (`build`).

**Review by WenInCode — CHANGES_REQUESTED** [review]
Minor suggestions :)

File: toolkit/package.json (hunk 1 above)
Comment (valeriia-ruban) [reply toolkit/package.json]: agree 👍 I thought a lot of the naming here 😅 

thank you for the ideas 💡

**Review by WenInCode — APPROVED** [review]
Looks great 👍

**Pull request merged**